src/main/webapp/app/modules/provider/all-records.scss

```diff
@@ -134,6 +134,25 @@
   }
 }
 
+.claim-exit-modal-bottom-divider {
```

Comment:
The `claim-exit-modal` sounds weird and is misleading. I think `exit-modal`  or `modal-on-exit` would be better.